README-build-from-source.md

```diff
@@ -1,194 +0,0 @@
-This document describes how to build ATF, UEFI, Linux kernel and Android for HiKey960 from source.
```

Comment:
One other idea here, is have a UEFI branch, which contains the build from source stuff - maybe as well as current UEFI binaries?  That way folks can tinker with the preview but master is clean?